tlsutil/config.go

```diff
@@ -920,11 +920,6 @@ func (c *Configurator) wrapALPNTLSClient(dc, nodeName, alpnProto string, conn ne
 		return nil, err
 	}
 
-	if cs := tlsConn.ConnectionState(); !cs.NegotiatedProtocolIsMutual {
```

Comment:
Just curious but why did this need removal?

Reply:
Deprecated in 1.15, it's always `true` now (and gave a compiler error when checking it) https://golang.org/pkg/crypto/tls/

Reply:
If I understand correctly, it would now fail to establish a connection at all if in the case wheree it would be false.

Reply:
:thumbsup: